### PR TITLE
Update automapper, update models

### PR DESCRIPTION
**Pull request by kaallaash**

### CODE REVIEW

File: DescriptiveGeometry/DG.API/ViewModels/ChangeDrawingViewModel.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public class ChangeDrawingViewModel
 {
-    public string Description { get; set; }
+    public string DescriptionText { get; set; }
```
Comment (nantonov) [inline DescriptiveGeometry/DG.API/ViewModels/ChangeDrawingViewModel.cs:5]: Why did you rename this prop?

File: DescriptiveGeometry/DG.API/ViewModels/DrawingViewModel.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 public class DrawingViewModel
 {
     public int Id { get; set; }
-    public string Description { get; set; }
+    public string DescriptionText { get; set; }
```
Comment (nantonov) [inline DescriptiveGeometry/DG.API/ViewModels/DrawingViewModel.cs:6]: Why did you rename this prop?

File: DescriptiveGeometry/DG.API/AutoMapper/MappingProfile.cs [hunk 3]
```diff
@@ -8,7 +8,51 @@ public class MappingProfile : Profile
 {
     public MappingProfile()
     {
-        CreateMap<Drawing, DrawingViewModel>().ReverseMap();
-        CreateMap<Drawing, ChangeDrawingViewModel>().ReverseMap();
+        CreateMap<DrawingViewModel, Drawing>()
```
Comment (nantonov) [inline DescriptiveGeometry/DG.API/AutoMapper/MappingProfile.cs:11]: Why it's so complex?
It's **auto**mapper

**Review by nantonov — CHANGES_REQUESTED** [review]

File: DescriptiveGeometry/DG.API/AutoMapper/MappingProfile.cs (hunk 3 above)
Comment (kaallaash) [reply DescriptiveGeometry/DG.API/AutoMapper/MappingProfile.cs]: Because I tried to map models with different properties and degree of nesting.
But it wasn't the best decision

File: DescriptiveGeometry/DG.API/ViewModels/DrawingViewModel.cs (hunk 2 above)
Comment (kaallaash) [reply DescriptiveGeometry/DG.API/ViewModels/DrawingViewModel.cs]: The answer is the same

File: DescriptiveGeometry/DG.API/ViewModels/ChangeDrawingViewModel.cs (hunk 1 above)
Comment (kaallaash) [reply DescriptiveGeometry/DG.API/ViewModels/ChangeDrawingViewModel.cs]: Because I had conflicts with automapper, which wanted to map description{string} to description{DrawingDescription}.
I tried to fool him with this action

**Review by Blackolf931 — APPROVED** [review]

**Review by nantonov — APPROVED** [review]
